File: Day326_2025/Q326_Leetcode.java
class Q326_Leetcode {
    public int minimumOperations(int[] nums) {
        int ans = 0;

        for (int v : nums) {
            int mod = v % 3;
            ans += Math.min(mod, 3 - mod);
        }
        return ans;
    }
}

File: Day326_2025/Q326_Leetcode.cpp
class Solution {
public:
    int minimumOperations(vector<int>& nums) {
        // Q. Find Minimum Operations to Make All Elements Divisible by Three
        return std::accumulate(nums.begin(), nums.end(), 0, [](int acc, int v) {
            return acc + std::min(v % 3, 3 - v % 3);
        });
    }
};